system-tests/lib/crecli/por/por.go

```diff
@@ -11,13 +11,24 @@ import (
 	libcrecli "github.com/smartcontractkit/chainlink/system-tests/lib/crecli"
 )
 
-func CreateConfigFile(feedsConsumerAddress common.Address, feedID, dataURL, writeTargetName string) (*os.File, error) {
+type ConfigFileInput struct {
+	FundedAddress        common.Address
+	BalanceReaderAddress common.Address
+	FeedsConsumerAddress common.Address
+	FeedID               string
+	DataURL              string
+	WriteTargetName      string
+	ReadTargetName       string
+	ExpectedFundAmount   string
```

Comment:
1. remove, not needed any more, right?
2. maybe add `json:",omitempty"` to all fields?